util/unifyfs/src/unifyfs-rm.c

```diff
@@ -483,58 +485,132 @@ static int slurm_read_resource(unifyfs_resource_t* resource)
 static size_t construct_server_argv(unifyfs_args_t* args,
                                     char** server_argv)
 {
-    size_t argc;
+    size_t argc = 0;
     char number[16];
 
```

Comment:
Add a single `if (server_argv != NULL)` check here so you don't have to do the check so many times within the function.